### DISCUSSION TITLE
parserOptions.project error

### DISCUSSION THREAD

**Discussion by LeoRedin** [infinitered/ignite#1851] [Q&A]
Hello everyone! I've just installed the project here and I am getting this annoying error, it does not block anything but is it related to some VS Code configuration maybe?

Most of the files that end in js have this same error.

![Screen Shot 2021-12-17 at 2 40 13 PM](https://user-images.githubusercontent.com/20828515/146585679-276da281-4d35-4904-a2f4-673ba450870e.png)

**Comment by jamonholmgren** [comment]
@LeoRedin Is this still happening with the latest Ignite?
